src/components/presentational/DumbbellChart/DumbbellChart.css

```diff
@@ -0,0 +1,103 @@
+.mdhui-dumbbell-visualization {
+    padding-top: 25px;
```

Comment:
This padding-top seems wrong based on the design.  The 250 should be right below the date range navigator.

Just removing it does cause a double border so you could also add margin-top:-1px or make the 250 tick just not have a border on it or something